code/implementation/docker/machine/src/main/resources/schema/base/amazonec2Config.json.d/amazonec2.json

```diff
@@ -12,8 +12,7 @@
         },
         "vpcId": {
             "type": "string",
```

Comment:
Setting both seems to be valid in the driver (if not terribly useful, since subnet implies vpc).  There should be validation on our side that one or the other is set though if possible instead of going off and failing in machine.